modules/integration/tests-common/test-utils/src/main/java/org/wso2/carbon/ml/integration/common/utils/MLHttpClient.java

```diff
@@ -322,6 +322,8 @@ public int getProjectId(String projectName) throws MLHttpClientException {
             throw new MLHttpClientException("Faile to get ID of project: " + projectName, e);
         } catch (IOException e) {
             throw new MLHttpClientException("Faile to get ID of project: " + projectName, e);
+        } catch (JSONException e) {
+            throw new MLHttpClientException("Faile to get ID of project: " + projectName, e);
```

Comment:
We're returning the same exception message here right? Hence, can we catch Exception only without catching all different exceptions? There's a typo in the message. /s/Faile/Failed